packages/web-worker/src/endpoint.ts

```diff
@@ -86,12 +92,14 @@ export interface Endpoint<T> {
   expose(api: {[key: string]: Function | undefined}): void;
   revoke(value: Function): void;
   exchange(value: Function, newValue: Function): void;
+  terminateEndpoint(): void;
```

Comment:
Named this to `terminateEndpoint` instead of `terminate` to avoid confusion between terminate calls

Reply:
Rename it somewhere else then, API should have the most obvious name which is definitely `terminate()` here